src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -147,7 +146,7 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
 
   protected async fetchEvalscriptUrlIfNeeded(): Promise<void> {
     if (this.evalscriptUrl && !this.evalscript) {
-      const response = await axios.get(this.evalscriptUrl, { responseType: 'text', useCache: true });
+      const response = await axios.get(this.evalscriptUrl, { responseType: 'text' });
```

Comment:
I know this is out of scope for this MR, but there is a bug here - `fetchEvalscriptUrlIfNeeded` does not use `reqConfig`. 

Because of this, removing `useCache` now changes the behaviour of the method. To fix this, `expiresIn` should be added. But maybe it is easier to just fix the bug?

Reply:
It was not a bug, I thought it would be best if consumers handle the caching, but this would be a breaking change.
The default behavior has been added back, and consumers can always disable caching on requests.

Reply:
I don't think we are talking about the same thing? I think the bug is in that user can't affect `axios.get` through `reqConfig`. 

Note that this is not a new bug. The reverting of default caching values now means that there is no change because of this MR, so if you feel this is out of scope, feel free to resolve.

It still needs to be fixed though... let me know if you resolve and I can add a task for this.

Reply:
Default caching has been added back, and users can override this behavior with `reqConfig`

Reply:
Do you think `fetchEvalscriptUrlIfNeeded` should use the same `reqConfig` as `getMap`?

Reply:
This has been fixed by passing `innerReqConfig` to `fetchEvalscriptUrlIfNeeded`